repotracker.ini

```diff
@@ -9,9 +9,12 @@ topic_prefix = VirtualTopic.eng.repotracker
 [datanommer]
 type = container
 repo = quay.io/factory2/datanommer
-tags = latest stage prod
 
 [datagrepper]
 type = container
 repo = quay.io/factory2/datagrepper
-tags = latest stage prod
+
+[secretrepo]
+type = container
+repo = quay.io/factory2/secret
+token = FACTORY2_QUAY_TOKEN
```

Comment:
Should this be `token_env`? That's the key referenced in `container.py`.

Reply:
yes, updated, thanks, also fixed tests and added new ones.

Reply:
Great!